scripts/data_ingestion/download_statcast_pitch_level.py

```
@@ -0,0 +1,145 @@
+        df = statcast(start_dt=start_date, end_dt=end_date)
+        
+        if df.empty:
+            print(f"  ⚠️  No Statcast data returned for {start_date} to {end_date}")
+            return False
+        
+        # Save to CSV
+        output_file = DATA_DIR / f"statcast_{start_date}_{end_date}.csv"
+        df.to_csv(output_file, index=False)
+        
+        print(f"  ✅ Downloaded {len(df)} Statcast rows to {output_file}")
+        return True
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)